test/specs/dist-default-types.spec.js

```diff
@@ -0,0 +1,31 @@
+const fs = require('fs-extra');
```

Comment:
Instead of creating a new spec file, you should just add 
```
    it('should copy default-types in dist', () => sandbox.generate()
        .then(({dir}) => fs.exists(`${dir}/dist/model/ts/node_modules/apikana/default-types.ts`, res => res || fail())))
```
to the existing apikana/test/specs/generate.spec.js file.